packages/now-cli/src/util/get-files.ts

```diff
@@ -183,7 +161,7 @@ export async function staticFiles(
 
     // Compile list of ignored patterns and files
     const ignoreName = isBuilds ? '.nowignore' : '.gitignore';
```

Comment:
I think `isBuilds` is always true since the comment says `true for Now 2.0 builders`. So can we remove some more logic in this function?

Reply:
Good catch 👍 
Removed it